### PR TITLE
Ports supermatter sprites from Baystation 12

### PR DESCRIPTION
**Pull request by quardbreak**
(cherry picked from commit 4654110894684822e6dd64bbc24caae5e19b760e)

## Description of changes

This pull request replaces supermatter sprite with current Baystation12 supermatter sprite. Difference with Baystation12 implementation is what supermatter now have basic crystal sprite and underlay plate sprite (which is separate)

## Why and what will this PR improve

It allows us to control color variations of supermatter and for future animate() delam color change.

## Authorship

Azlan, https://github.com/Baystation12/Baystation12/pull/30803
@listerla, https://github.com/NebulaSS13/Nebula/pull/1851
@MistakeNot4892, for a 32x32 resized sprite.

## Changelog

:cl: Azlan (as the sprite author) and quardbreak
imageadd: Replaced Supermatter sprites with ones by Azlan.
/:cl:

### CODE REVIEW

**Comment by quardbreak** [comment]
ToDo: address feedback comment https://github.com/NebulaSS13/Nebula/pull/1851#discussion_r666372948

**Comment by MistakeNot4892** [comment]
Should also note you ganked this from limette's PR :P

**Comment by quardbreak** [comment]
CherryPick is linked on top by the way :D

**Comment by MistakeNot4892** [comment]
And now listerla's PR is linked too.

**Comment by listerla** [comment]
supermatter_shard_underplate seems to be an animated icon with a state where it includes the glowing shard.

**Comment by quardbreak** [comment]
@listerla Good gotcha, thanks!

File: code/modules/supermatter/supermatter.dm [hunk 1]
```diff
@@ -84,12 +84,12 @@ var/global/list/supermatter_final_thoughts = list(
...
 	desc = "A strangely translucent and iridescent crystal. <span class='danger'>You get headaches just from looking at it.</span>"
-	icon = 'icons/obj/engine.dmi'
-	icon_state = "darkmatter"
+	icon = 'icons/obj/supermatter.dmi'
+	icon_state = "supermatter"
 	density = 1
 	anchored = 0
 	light_range = 4
-	layer = ABOVE_OBJ_LAYER
+	layer = ABOVE_HUMAN_LAYER
```
Comment (afterthought2) [inline code/modules/supermatter/supermatter.dm:92]: I mean, I still hate this. Would much rather have the sprite be the correct size instead.

Comment (Kurgis) [reply code/modules/supermatter/supermatter.dm:92]: My main issue with this comment is that the complaint is based around a false pretense, there's other objects in the game which are larger than 32x32. They are just also set on the incorrect `ABOVE_OBJ_LAYER` so they phase above Human mobs, so you just didn't notice them searching the code.

Comment (MistakeNot4892) [reply code/modules/supermatter/supermatter.dm:92]: It's not a false pretense to say that he dislikes it and would prefer if they were 32x32. There are not many larger icons in the game and the ones that are in general circulation tend to either be problematic (multitile airlocks) or inaccessible (RUST cores).

Comment (afterthought2) [reply code/modules/supermatter/supermatter.dm:92]: I believe in a similar comment on the earlier PR, I said other objs don't do this: I stand corrected, but my overall sentiment remains unchanged.

**Review by MistakeNot4892 — DISMISSED** [review]
After some discussion on Discord, imo this is fine to go in. If we decide down the track that bigger-than-32x32 icons are to be avoided, it can be reworked in a proper pass.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**